apps/expo/.env.development

```diff
@@ -25,3 +25,4 @@ SENTRY_SERVER_ENVIRONMENT=local
 SENTRY_SERVER_RELEASE_ENVIRONMENT=local
 
 NEXT_PUBLIC_META_SINGLE_EDITION_MINTABLE_CREATOR=0x50C001A33Caa446c8b84C489F371F77754F41024
+NEXT_PUBLIC_WEBSITE_DOMAIN=dev.showtime.xyz
```

Comment:
we need to point `dev` CNAME to vercel.


<img width="1145" alt="Screenshot 2022-10-03 at 5 02 25 PM" src="https://user-images.githubusercontent.com/23293248/193567126-3bb1c379-6866-43f7-8733-6b77c0b0cc15.png">